apps/antalmanac/src/components/RightPane/SectionTable/SectionTableBody/SectionTableBodyCells/LocationsCell.tsx

```diff
@@ -1,26 +1,18 @@
 import { Box } from '@mui/material';
 import { WebsocSectionMeeting } from '@packages/antalmanac-types';
-import { Fragment, useCallback } from 'react';
-import { Link } from 'react-router-dom';
+import { Fragment } from 'react';
+
+import { MapLink } from '../../../../buttons/MapLink';
```

Comment:
```suggestion
import { MapLink } from '$components/buttons/MapLink';
```
repeat: prefer import aliases (notice here how the deeply nested access of mapLink is rather unwieldy and unportable?)